omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -197,7 +197,7 @@
            {ip_mode, [ <%=PrivateChef['use_ipv6'] ? "ipv6,ipv4" : "ipv4" %> ] },
            %% Database connection parameters
            {db_host, "<%= node['private_chef']['postgresql']['vip'] %>"},
-           {db_port, 5432},
+           {db_port, <%= node['private_chef']['opscode-erchef']['sql_user'] %>},
```

Comment:
I imagine this should be `<%= node['private_chef']['postgresql']['port'] %>` rather than sql_user?

An aside: We should probably (eventually) always have 2 attributes 1 that controls what port the service listens on when we set it up, and another to control what port applications should talk to.  By default they would be the same, but it would give us the flexibility to add connection pooling/tpc proxies such that erchef would actually talk to a different port.